packages/error-view/index.tsx

```diff
@@ -0,0 +1,40 @@
+import s from './style.module.css'
+import Link from 'next/link'
+import useErrorPageAnalytics from './use-error-page-analytics'
+
+interface ErrorPageProps {
+  /** Error code to be recorded via window.analytics.track.  */
+  statusCode: number
+}
+
+const CONTENT_DICT = {
```

Comment:
Went with a very fixed content approach for this component. I think this will satisfy most of our existing use cases, and have provided the `useErrorPageAnalytics` in case we need to do something more custom (in terms of either content or styles).